recipe/meta.yaml

```diff
@@ -46,7 +46,10 @@ test:
     - shapely.vectorized
   commands:
     - pip check
-    - pytest --pyargs shapely
+    # osx fails if LC_ALL is not Italian or Portuguese
+    # see https://github.com/shapely/shapely/blob/2.0.5/shapely/tests/legacy/test_locale.py#L14
+    - export LC_ALL=it_IT.UTF-8  # [osx]
+    - pytest shapely/tests -v
```

Comment:
You should be able to remove the source_files section and run `pytest --pyargs shapely`, as the tests are included in the package itself. This avoids having a copy of the tests in the info folder.

Reply:
How are you able to tell if the tests are included in the package itself?
I end up using `source_files` a lot, and am now wondering if there are times I actually don't need to.

Reply:
Here I see them nested in the shapely package: https://github.com/shapely/shapely/tree/2.0.5/shapely/tests